### PR TITLE
#116

### PR DESCRIPTION
**Pull request by jkeung**
changes for learning curve viz. created new branch with tests

### CODE REVIEW

**Comment by bbengfort** [comment]
@jkeung I'm sorry it's taken me so long to get to this - I'm planning to get to it on Wednesday.

File: tests/test_classifier/test_learning_curve.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        y = self.occupancy['occupancy'].astype(int)
+
+        X = X.view((float, len(X.dtype.names)))
+        
+        with self.assertRaises(YellowbrickError):
+            visualizer = LearningCurveVisualizer(LinearSVC(), 
+                train_sizes=10000,
+                cv=ShuffleSplit(n_splits=100, test_size=0.2, random_state=0))
+            visualizer.fit(X, y)
+            visualizer.poof()
```
Comment (ndanielsen) [inline tests/test_classifier/test_learning_curve.py]: I'm getting an IndexError when I run this locally.

======================================================================
ERROR: Test learning curve with bad input for training size.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/ndanielsen/ddl/jyb/tests/test_classifier/test_learning_curve.py", line 150, in test_learning_curve_bad_trainsize
    visualizer.fit(X, y)
  File "/Users/ndanielsen/ddl/jyb/yellowbrick/classifier/learning_curve.py", line 116, in fit
    self.estimator, X, y, cv=self.cv, n_jobs=self.n_jobs, train_sizes=self.train_sizes)
  File "/usr/local/var/pyenv/versions/yb-dev/lib/python3.5/site-packages/sklearn/model_selection/_validation.py", line 756, in learning_curve
    n_max_training_samples)
  File "/usr/local/var/pyenv/versions/yb-dev/lib/python3.5/site-packages/sklearn/model_selection/_validation.py", line 808, in _translate_train_sizes
    n_ticks = train_sizes_abs.shape[0]
IndexError: tuple index out of range

File: tests/test_classifier/test_learning_curve.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        except Exception as e:
+            self.fail("error during learning curve: {}".format(e))
+
+    def test_learning_curve_bad_trainsize(self):
+        """
+        Test learning curve with bad input for training size.
+        """
+
+        X = self.occupancy[[
```
Comment (ndanielsen) [inline tests/test_classifier/test_learning_curve.py]: I noticed that you are testing this on the entire data set, I recommend that for speeding these tests up you slice and test the first 1,000 rows or something like that.

**Comment by bbengfort** [comment]
@jkeung also - I'm getting an `ImageNotClose` error when running the tests -- can you look at the image diff locally?

**Pull request closed without merging**